umap/static/umap/js/modules/rendering/ui.js

```diff
@@ -380,8 +385,16 @@ export const LeafletPolyline = Polyline.extend({
   },
 
   getMeasure: function (shape) {
+    const shapes = shape
```

Comment:
That section is not trivial to readerstand

Reply:
Agreed! I've changed it :)